packages/transformers/html/src/dependencies.js

```diff
@@ -142,10 +142,20 @@ export default function collectDependencies(
       (attrs.rel === 'canonical' || attrs.rel === 'manifest') &&
       attrs.href
     ) {
-      attrs.href = asset.addURLDependency(attrs.href, {
+      let href = attrs.href;
+      if (attrs.rel === 'manifest') {
+        // A hack to allow manifest.json rather than manifest.webmanifest.
+        // If a custom pipeline is used, it is responsible for running @parcel/transformer-webmanifest.
+        if (!href.includes(':')) {
+          href = 'webmanifest:' + href;
```

Comment:
This could be problematic when someone redefines this pipeline in their local parcelrc and doesn't include the webmanifest transformer, but oh well